VultisigApp/VultisigApp/Views/Keygen/KeygenView.swift

```diff
@@ -198,6 +206,15 @@ struct KeygenView: View {
             oldResharePrefix: oldResharePrefix
         )
     }
+    
+    private func setDoneData() {
+        if tssType == .Reshare {
+            vault.isBackedUp = true
```

Comment:
I think this is a misunderstanding , after reshare , we actually want user to backup their vault

Reply:
set `vault.isBackupUp = false`

Reply:
Yes, after import = no banner.
after reshare = banner